"./build.py build" doesn't work on MacOSX environment

The error message was as follows.

<pre>
(buildpy)MBP2012:ol3-git sanak$ ./build.py build
2014-10-13 21:30:17,678 build/ol-debug.js: node tasks/build.js config/ol-debug.json build/ol-debug.js
info ol Parsing dependencies
info ol No compile options found.  Concatenating 333 sources
ERR! Trouble concatenating sources.  EMFILE, open '/Users/sanak/Build/js/ol3-git/src/ol/format/igcformat.js' 

module.js:340
    throw err;
          ^
Error: Cannot find module 'rimraf'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at cleanupDirs (/Users/sanak/Build/js/ol3-git/node_modules/temp/lib/temp.js:96:20)
    at process.cleanup (/Users/sanak/Build/js/ol3-git/node_modules/temp/lib/temp.js:119:19)
    at process.emit (events.js:95:17)
    at process.exit (node.js:711:17)
    at module.exports (/Users/sanak/Build/js/ol3-git/tasks/build.js:266:15)
    at /Users/sanak/Build/js/ol3-git/node_modules/async/lib/async.js:485:30
2014-10-13 21:30:21,256 build/ol-debug.js: clean
2014-10-13 21:30:21,256 pake: build/ol-debug.js: Command '['node', 'tasks/build.js', 'config/ol-debug.json', 'build/ol-debug.js']' returned non-zero exit status 8
</pre>

--- a/package.json
+++ b/package.json
@@ -17,25 +17,25 @@
     "url": "https://github.com/openlayers/ol3/issues"
   },
   "dependencies": {
-    "async": "~0.2.10",
-    "closure-util": "~1.0.0",
-    "fs-extra": "~0.8.1",
-    "graceful-fs": "~3.0.2",
-    "htmlparser2": "~3.7.1",
-    "jsdoc": "~3.3.0-alpha9",
-    "nomnom": "~1.6.2",
-    "temp": "~0.7.0",
-    "walk": "~2.3.3"
+    "async": "0.2.10",
+    "closure-util": "1.0.0",
+    "fs-extra": "0.8.1",
+    "graceful-fs": "3.0.2",
+    "htmlparser2": "3.7.1",
+    "jsdoc": "3.3.0-alpha9",
+    "nomnom": "1.6.2",
+    "temp": "0.7.0",
+    "walk": "2.3.3"
   },
   "devDependencies": {
-    "clean-css": "^2.2.7",
-    "expect.js": "~0.3.1",
-    "jquery": "~2.1.1",
-    "jshint": "~2.5.1",
-    "mocha": "~1.20.1",
-    "mocha-phantomjs": "~3.5.0",
-    "phantomjs": "~1.9.7-5",
-    "proj4": "~2.2.1",
-    "sinon": "~1.10.2"
+    "clean-css": "2.2.7",
+    "expect.js": "0.3.1",
+    "jquery": "2.1.1",
+    "jshint": "2.5.1",
+    "mocha": "1.20.1",
+    "mocha-phantomjs": "3.5.0",
+    "phantomjs": "1.9.10",
+    "proj4": "2.2.1",
+    "sinon": "1.10.2"
   }
 }

